Custom default docker daemon config for LMP

LMP specific docker daemon config, which can be replaced by the user by
replacing the standard systemd service file in /etc/systemd/system.

Upstream-Status: Inappropriate [lmp specific]

Signed-off-by: Ricardo Salveti <ricardo@foundries.io>

Index: git/src/import/contrib/init/systemd/docker.service
===================================================================
--- git.orig/src/import/contrib/init/systemd/docker.service
+++ git/src/import/contrib/init/systemd/docker.service
@@ -10,7 +10,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd://
+ExecStart=/usr/bin/dockerd -H fd:// --config-file /usr/lib/docker/daemon.json
 ExecReload=/bin/kill -s HUP $MAINPID
 LimitNOFILE=1048576
 # Having non-zero Limit*s causes performance problems due to accounting overhead
